tools/Utilities/src/Strings/en-us/Resources.resw

```diff
@@ -136,7 +135,7 @@
     <comment>Utilities</comment>
   </data>
   <data name="RegistryPreviewUtilityDesc" xml:space="preserve">
-    <value>A quick little utility to visualize and edit complex Windows Registry files.</value>
+    <value>Edit and visualize Windows Registry files faster.</value>

```

Comment:
Compared to what?  RegEdit?

Reply:
I think so.

Reply:
I think the meta point here is that the original descriptions seem better to me. Maybe without the "A quick little" or "Quick and simple". I know content writers have had a look at this but I would think "Visualize and edit complex Windows Registry files." would be enough here. Same for host file, e.g. "Managing hosts file; add, filter, or remove file entries", something like that.

Reply:
"Utility to edit and visualize registry files" ?

cc: @cinnamon-msft

Reply:
updated.